attachment_preview/static/src/js/attachment_preview.js

```diff
@@ -208,22 +211,21 @@ openerp.attachment_preview = function(instance)
                 {
                     _(extensions).each(function(extension)
                     {
-                        var $element = self.$el.find('.oe-binary-preview');
-                        if(openerp.attachment_preview.can_preview(extension))
+                        var $element = self.$('.oe_binary_preview');
+                        $element.unbind( "click" );
+                       if(can_preview(extension[0]))
                         {
                             $element.click(function()
                             {
-                                openerp.attachment_preview.show_preview(
+                                show_preview(
                                     null,
-                                    _.str.sprintf(
-                                        '/web/binary/saveas?session_id=%s&model=%s&field=%s&id=%d',
-                                        instance.session.session_id,
-                                        self.view.dataset.model,
-                                        self.name,
-                                        self.view.datarecord.id),
-                                    extension,
+
+                                        ('data:application/'+extension[0]+';base64,'+
```

Comment:
why would you want to do this? With a few really big attachments, your browser becomes really inresponsive